<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AWidget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>650</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btnReady">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>400</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>准备开始</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>200</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>迷你简硬笔行书</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>斗地主游戏</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>-1</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="btnForLord">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>叫</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNotLord">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>不叫</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBeat">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>出牌</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNotBeat">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>不出</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>421</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hLayout" stretch="2,1,2,1,2">
      <item>
       <widget class="QLabel" name="bottomL">
        <property name="font">
         <font>
          <family>迷你简硬笔行书</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>底牌1</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="bottomM">
        <property name="font">
         <font>
          <family>迷你简硬笔行书</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>底牌2</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="bottomR">
        <property name="font">
         <font>
          <family>迷你简硬笔行书</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>底牌3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>140</y>
       <width>141</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="bvLayout" stretch="5,1">
      <item>
       <widget class="QLabel" name="bRole">
        <property name="font">
         <font>
          <family>迷你简硬笔行书</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>玩家2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bCount">
        <property name="font">
         <font>
          <family>迷你简硬笔行书</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>141</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="cvLayout" stretch="5,1">
      <item>
       <widget class="QLabel" name="cRole">
        <property name="font">
         <font>
          <family>迷你简硬笔行书</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>玩家1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cCount">
        <property name="font">
         <font>
          <family>迷你简硬笔行书</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>500</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="avLayout" stretch="3,1">
      <item>
       <widget class="QLabel" name="aRole">
        <property name="font">
         <font>
          <family>迷你简硬笔行书</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>本人</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="aCount">
        <property name="font">
         <font>
          <family>迷你简硬笔行书</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>350</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>351</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>350</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>again</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>350</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>quit</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
